run_pacta_data_preparation.R

```diff
@@ -103,6 +103,16 @@ logger::log_info(
 
 scenario_raw_data_to_include <- lapply(scenario_raw_data_to_include, get, envir = asNamespace("pacta.scenario.preparation"))
 
+factset_timestamp <-
+  unique(sub("_factset_.*[.]rds$", "", c(
```

Comment:
Style question: I've noticed you prefer using groups to denote literal `.`, rather than escaping (`[.]` vs `\\.`). Is there a reason for this?

I don't think we're particularly worried about optimizing our regexes, but I believe there is a small performance hit for groups over escaped characters.

Reply:
Definitely just a personal preference. I find it easier to read and harder to screw up than managing doubled backslashes to avoid R string escapes. Something I picked up from maintaining the regexes in countrycode.

Reply:
e.g. one can copy-paste this regex into a regex validator, but one with a R-style backslash won't work the same